apps/web/src/pages/ApplicantDashboardPage/components/ReviewRecruitmentProcessDialog.tsx

```diff
@@ -0,0 +1,416 @@
+  isSuspended: "true" | "false";
+}
+
+const ReviewRecruitmentProcess_Mutation = graphql(/* GraphQL */ `
+  mutation Name($id: ID!, $isSuspended: Boolean!) {
+    changeApplicationSuspendedAt(id: $id, isSuspended: $isSuspended) {
+      id
+      suspendedAt
+    }
+  }
+`);
+
+const ReviewRecruitmentProcessDialog_Fragment = graphql(/* GraphQL */ `
+  fragment ReviewRecruitmentProcessDialog on PoolCandidate {
+    id
+    expiryDate
+    finalDecisionAt
+    suspendedAt
+    placedAt
+    pool {
+      id
+      name {
+        localized
+      }
+      classification {
+        group
+        level
+        minSalary
+        maxSalary
+      }
+      workStream {
+        name {
+          localized
+        }
+      }
+      language {
+        value
+        label {
+          localized
+        }
+      }
+      department {
+        name {
+          localized
+        }
+      }
+      opportunityLength {
+        value
+        label {
+          localized
+        }
+      }
+      isRemote
+      location {
+        localized
+      }
+      securityClearance {
+        value
+        label {
+          localized
+        }
+      }
+      processNumber
+    }
+  }
+`);
+
+interface ReviewRecruitmentProcessDialogProps {
+  recruitmentProcessQuery: FragmentType<
+    typeof ReviewRecruitmentProcessDialog_Fragment
+  >;
+}
+
+const ReviewRecruitmentProcessDialog = ({
+  recruitmentProcessQuery,
+}: ReviewRecruitmentProcessDialogProps) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const paths = useRoutes();
+
+  const recruitmentProcess = getFragment(
+    ReviewRecruitmentProcessDialog_Fragment,
+    recruitmentProcessQuery,
+  );
+
+  const pool = recruitmentProcess?.pool;
+
+  const nullMessage = intl.formatMessage(commonMessages.notFound);
+  const poolName =
+    pool.name?.localized ?? intl.formatMessage(commonMessages.notFound);
+
+  const [, executeMutation] = useMutation(ReviewRecruitmentProcess_Mutation);
+
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const isSuspended = !!recruitmentProcess.suspendedAt;
+  const methods = useForm<FormValues>({
+    defaultValues: { isSuspended: isSuspended ? "true" : "false" },
+  });
+  const {
+    handleSubmit,
+    formState: { isSubmitting },
+  } = methods;
+
+  const updateSuspendedAtStatus = async (values: FormValues) => {
+    await executeMutation({
+      id: recruitmentProcess.id,
+      isSuspended: values.isSuspended === "true",
+    })
+      .then((res) => {
+        if (!res.error) {
+          setIsOpen(false);
+          if (values.isSuspended === "true") {
+            toast.success(
+              intl.formatMessage({
+                defaultMessage:
+                  "You have been removed from the search results.",
+                id: "PoFTwr",
+                description:
+                  "Alert displayed to the user when application card dialog submits successfully.",
+              }),
+            );
```

Comment:
Since you can update multiple different items on this page, I tihnk we need to be specific in the toast. "...remove from search results for {title}."

Reply:
@NienkeBr is the current copy okay? Or should we change it to what Eric suggested?

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/issues/12699